aws/terraform/infra/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "app" {
+  default     = "metaflow-infra"
```

Comment:
No need for type?

Reply:
Correct. `type` is for when you define variables.